Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common
 # install Ruby from source
 #  from source: mainly because of GEM native extensions,
 #  this is the most reliable way to use Ruby no Ubuntu if GEM native extensions are required
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev libsqlite3-dev
```

Comment:
Just tested:

```
$ apt-get install libsqlite3-dev

Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  libc-dev-bin libc6-dev libsqlite3-0 linux-libc-dev manpages manpages-dev
Suggested packages:
  glibc-doc sqlite3-doc man-browser
The following NEW packages will be installed:
  libc-dev-bin libc6-dev libsqlite3-0 libsqlite3-dev linux-libc-dev manpages manpages-dev
0 upgraded, 7 newly installed, 0 to remove and 7 not upgraded.
```

So it seems it does install the `libsqlite3-0` package, but not `sqlite3`.

That said, the point of the `system_report.sh` is to help _you_, to see when a pre-installed package is updated or when you want to know which version is installed, so I would still prefer to have at least some kind of version print for the package, if possible.

WDYT @bootstraponline ?